.gitignore

```diff
@@ -33,4 +33,7 @@ crates/prover/pk
 crates/prover/powersOfTau28_hez_final.ptau
 crates/prover/semaphore-gnark-11
 crates/prover/trusted-setup
-crates/prover/vk
\ No newline at end of file
+crates/prover/vk
+
+# Example legacy elf
+examples/elf 
```

Comment:
When jonathan made !include_elf, he also manually copied the elf from /target to the workspace root, to maintain backward compatibility with the old !include_bytes method. Let's gitignore that elf in examples.

Reply:
Should we gitignore the cargo.lock in examples too?